db/liquibase/changesets/20180917_entity.xml

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <databaseChangeLog logicalFilePath="avram" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
-    <changeSet logicalFilePath="avram" author="ansingh" id="add_entity_table">
+    <changeSet author="ansingh" id="create_entity_table">
         <createTable tableName="entity">
             <column name="id" type="bigserial">
                 <constraints primaryKey="true"/>
             </column>
-            <column name="name" type="varchar(1000)"/>
-            <column name="collection_id" type="bigserial">
+            <column name="external_id" type="UUID">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="external_collection_id" type="UUID">
```

Comment:
We might want to use `collection_id` as the foreign key. I don't have a solid reason for it other than my understanding of data modeling practices. `id` is the primary key while `external_id` is an alternate key. However, using the primary key means having a join on `collection` when selecting a collection's entities by `external_collection_id`.

I'm okay with keeping it like this to get this PR merged and thinking on it more before going to production.